### PR TITLE
ETC, ITEM, QUEST_JOBSTEP, SKILL - "Merog"s and Bramble

### PR DESCRIPTION
**Pull request by vtange**
Standardize name of common mob. Merlog/Merrogg -> Merog.

Special thanks to g1real.

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
-QUEST_JOBSTEP_20150317_001334	The Ranger Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001335	The Ranger Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001336	Collect rocks from the area the Ranger Master mentioned
-QUEST_JOBSTEP_20150317_001337	Collected all the books for the Ranger Master. Talk to the Ranger Master to be acknowledged as a Ranger.
+QUEST_JOBSTEP_20150317_001331	This isn't the only book Ranger Master is looking for. Talk to the Ranger Master again.
+QUEST_JOBSTEP_20150317_001332	Hunter Master has the book Ranger Master wants. Prepare 5000 silvers to get the book from Hunter Master.
+QUEST_JOBSTEP_20150317_001333	Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
+QUEST_JOBSTEP_20150317_001334	Ranger Master told you that you may be able to trade with Quarrel Shooter Master with the leathers of Merogs for the book. Obtain the leathers of Merogs from the Fallen Worry Forest.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Ranger Master -> The Ranger Master

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
-QUEST_JOBSTEP_20150317_001333	The Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
-QUEST_JOBSTEP_20150317_001334	The Ranger Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001335	The Ranger Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001336	Collect rocks from the area the Ranger Master mentioned
-QUEST_JOBSTEP_20150317_001337	Collected all the books for the Ranger Master. Talk to the Ranger Master to be acknowledged as a Ranger.
+QUEST_JOBSTEP_20150317_001331	This isn't the only book Ranger Master is looking for. Talk to the Ranger Master again.
+QUEST_JOBSTEP_20150317_001332	Hunter Master has the book Ranger Master wants. Prepare 5000 silvers to get the book from Hunter Master.
+QUEST_JOBSTEP_20150317_001333	Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Archer Master -> The Archer Master

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
-QUEST_JOBSTEP_20150317_001332	The Hunter Master has the book the Ranger Master wants. Prepare 5000 silvers to get the book from the Hunter Master.
-QUEST_JOBSTEP_20150317_001333	The Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
-QUEST_JOBSTEP_20150317_001334	The Ranger Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001335	The Ranger Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001336	Collect rocks from the area the Ranger Master mentioned
-QUEST_JOBSTEP_20150317_001337	Collected all the books for the Ranger Master. Talk to the Ranger Master to be acknowledged as a Ranger.
+QUEST_JOBSTEP_20150317_001331	This isn't the only book Ranger Master is looking for. Talk to the Ranger Master again.
+QUEST_JOBSTEP_20150317_001332	Hunter Master has the book Ranger Master wants. Prepare 5000 silvers to get the book from Hunter Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Hunter Master -> The Hunter Master

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
 QUEST_JOBSTEP_20150317_001330	Get Lydia Schaffen's book from Vubbe Fighter
-QUEST_JOBSTEP_20150317_001331	This isn't the only book the Ranger Master is looking for. Talk to the Ranger Master again.
-QUEST_JOBSTEP_20150317_001332	The Hunter Master has the book the Ranger Master wants. Prepare 5000 silvers to get the book from the Hunter Master.
-QUEST_JOBSTEP_20150317_001333	The Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
-QUEST_JOBSTEP_20150317_001334	The Ranger Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001335	The Ranger Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001336	Collect rocks from the area the Ranger Master mentioned
-QUEST_JOBSTEP_20150317_001337	Collected all the books for the Ranger Master. Talk to the Ranger Master to be acknowledged as a Ranger.
+QUEST_JOBSTEP_20150317_001331	This isn't the only book Ranger Master is looking for. Talk to the Ranger Master again.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: This isn't the only book Ranger Master is looking for -> This isn't the only book that the Ranger Master is looking for

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
+QUEST_JOBSTEP_20150317_001332	Hunter Master has the book Ranger Master wants. Prepare 5000 silvers to get the book from Hunter Master.
+QUEST_JOBSTEP_20150317_001333	Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
+QUEST_JOBSTEP_20150317_001334	Ranger Master told you that you may be able to trade with Quarrel Shooter Master with the leathers of Merogs for the book. Obtain the leathers of Merogs from the Fallen Worry Forest.
+QUEST_JOBSTEP_20150317_001335	Ranger Master looks very happy to have found the book. Talk to him to find the next book.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Ranger Master -> The Ranger Master

File: QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
...
+QUEST_JOBSTEP_20150317_001336	Collect rocks from the area Ranger Master mentioned
+QUEST_JOBSTEP_20150317_001337	Collected all the books for Ranger Master. Talk to Ranger Master to be acknowledged as a Ranger.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Ranger Master -> the Ranger Master

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -1311,13 +1311,13 @@ QUEST_JOBSTEP_20150317_001310	Go to the Archer Master in Klaipeda.
...
-QUEST_JOBSTEP_20150317_001317	Gathered enough money for the book. Give 5000 silvers to the Hunter Master in exchange for the book and take the book to the Archer Master.
-QUEST_JOBSTEP_20150317_001318	The Archer Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001319	Collect the Merrogg Leather
-QUEST_JOBSTEP_20150317_001320	The Archer Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather you have for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
+QUEST_JOBSTEP_20150317_001314	Found the book Archer Master is looking for. Take it to Archer Master.
+QUEST_JOBSTEP_20150317_001315	This isn't the only book Archer Master is looking for. Talk to the Archer Master again.
+QUEST_JOBSTEP_20150317_001316	Hunter Master has the book Archer Master wants. Prepare 5000 silvers to get the book from Hunter Master.
+QUEST_JOBSTEP_20150317_001317	Gathered enough money for the book. Give 5000 silvers to Hunter Master in exchange for the book and take the book to Archer Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Hunter Master -> the Hunter Master
Archer Master -> the Archer Master

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -1311,13 +1311,13 @@ QUEST_JOBSTEP_20150317_001310	Go to the Archer Master in Klaipeda.
...
 QUEST_JOBSTEP_20150317_001313	Deliver book to Master Archer
-QUEST_JOBSTEP_20150317_001314	Found the book the Archer Master is looking for. Take it to the Archer Master.
-QUEST_JOBSTEP_20150317_001315	This isn't the only book the Archer Master is looking for. Talk to the Archer Master again.
-QUEST_JOBSTEP_20150317_001316	The Hunter Master has the book the Archer Master wants. Prepare 5000 silvers to get the book from the Hunter Master.
-QUEST_JOBSTEP_20150317_001317	Gathered enough money for the book. Give 5000 silvers to the Hunter Master in exchange for the book and take the book to the Archer Master.
-QUEST_JOBSTEP_20150317_001318	The Archer Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001319	Collect the Merrogg Leather
-QUEST_JOBSTEP_20150317_001320	The Archer Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather you have for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
+QUEST_JOBSTEP_20150317_001314	Found the book Archer Master is looking for. Take it to Archer Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Found the book Archer Master is looking for. Take it to Archer Master. -> Found the book that the Archer Master is looking for. Take it to the Archer Master.

File: QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -1311,13 +1311,13 @@ QUEST_JOBSTEP_20150317_001310	Go to the Archer Master in Klaipeda.
...
+QUEST_JOBSTEP_20150317_001315	This isn't the only book Archer Master is looking for. Talk to the Archer Master again.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: This isn't the only book Archer Master is looking for -> This isn't the only book that the Archer Master is looking for

File: QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -1311,13 +1311,13 @@ QUEST_JOBSTEP_20150317_001310	Go to the Archer Master in Klaipeda.
...
+QUEST_JOBSTEP_20150317_001316	Hunter Master has the book Archer Master wants. Prepare 5000 silvers to get the book from Hunter Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Hunter Master has the book Archer Master wants. Prepare 5000 silvers to get the book from Hunter Master. -> The Hunter Master has the book that the Archer Master wants. Prepare 5000 silvers to get the book from the Hunter Master.

File: QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -1274,13 +1274,13 @@ QUEST_JOBSTEP_20150317_001273	The Hunter Master is looking for a book, and he sa
...
-QUEST_JOBSTEP_20150317_001281	You've obtained enough Merrogg Leather that are needed by the Quarrel Shooter Master. Hand over the Merrogg Leather to the Quarrel Shooter Master at Klaipeda.
+QUEST_JOBSTEP_20150317_001277	This isn't the only book the Hunter Master is looking for. Talk to him again.
+QUEST_JOBSTEP_20150317_001278	Collect Merog Leather
+QUEST_JOBSTEP_20150317_001279	The Hunter Master told you that you may be able to trade with the Quarrel Shooter Master with the Merog leather you have for the book. Obtain the Merog leather from the Fallen Worry Forest.
+QUEST_JOBSTEP_20150317_001280	Hand over the Merog leather to the Quarrel Shooter Master
+QUEST_JOBSTEP_20150317_001281	You've obtained enough Merog leather for the Quarrel Shooter Master. Hand it over to the Quarrel Shooter Master at Klaipeda.
 QUEST_JOBSTEP_20150317_001282	Obtain %s by defeating Merogs
-QUEST_JOBSTEP_20150317_001283	The Hunter Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
+QUEST_JOBSTEP_20150317_001283	Hunter Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Hunter Master -> The Hunter Master

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**